src/khoj/processor/github/github_to_jsonl.py

```diff
@@ -34,26 +34,42 @@ def wait_for_rate_limit_reset(response, func, *args, **kwargs):
             return
 
     def process(self, previous_entries=None):
+        current_entries = []
+        for repo in self.config.repos:
+            current_entries += self.process_repo(repo, previous_entries)
+
+        return self.update_entries_with_ids(current_entries, previous_entries)
+
+    def process_repo(self, repo: GithubRepoConfig, previous_entries=None):
+        repo_url = f"https://api.github.com/repos/{repo.owner}/{repo.name}"
+        logger.info(f"Processing github repo {repo.owner}/{repo.name}")
```

Comment:
Maybe just put `f"{repo.owner}/{repo.name}"` into a separate variable as it is used everywhere in the function?
```suggestion
        repo_id = f'{repo.owner}/{repo.name}"
        repo_url = f"https://api.github.com/repos/{repo_id}"
        logger.info(f"Processing github repo {repo_id}")
```